velox/exec/Task.cpp

```diff
@@ -1411,7 +1411,28 @@ void Task::addOperatorStats(OperatorStats& stats) {
   taskStats_.pipelineStats[stats.pipelineId]
       .operatorStats[stats.operatorId]
       .add(stats);
-  stats.clear();
+}
+
+TaskStats Task::taskStats() const {
+  std::lock_guard<std::mutex> l(mutex_);
+
+  // This will contain task stats plust stats for the completed operators.
```

Comment:
```
nit: completed drivers but not completed operators?
// 'taskStats_' contains the stats for the completed drivers.
```

Reply:
Well, driver contains operators and we are actually interested in operators' stats per se. Drivers don't even have any stats. So can turn it either way. I can change if you'll insist.

Reply:
I have updated comments with "drivers (their operators)" instead of "operators".